dm/pkg/checker/lightning.go

```diff
@@ -205,40 +205,40 @@ func (c *LightningFreeSpaceChecker) Check(ctx context.Context) *Result {
 	return result
 }
 
-// LightningSortingSpaceChecker checks the local disk has enough space for physical
-// import mode sorting data.
-type LightningSortingSpaceChecker struct {
+// LightningCDCPiTRChecker checks whether the cluster has running CDC PiTR tasks.
+type LightningCDCPiTRChecker struct {
 	inner restore.PrecheckItem
 }
 
-// NewLightningSortingSpaceChecker creates a new LightningSortingSpaceChecker.
-func NewLightningSortingSpaceChecker(lightningChecker restore.PrecheckItem) RealChecker {
-	return &LightningSortingSpaceChecker{inner: lightningChecker}
+// NewLightningCDCPiTRChecker creates a new LightningClusterVersionChecker.
+func NewLightningCDCPiTRChecker(lightningChecker restore.PrecheckItem) RealChecker {
+	c, ok := lightningChecker.(*restore.CDCPITRCheckItem)
+	if ok {
+		c.Instruction = "physical import mode is not compatible with them. Please switch to logical import mode then try again."
```

Comment:
```suggestion
		c.Instruction = "there are running CDC or PiTR, physical import mode is not compatible with them. Please switch to logical import mode then try again."
```

Reply:
The detailed message (CDC or PiTR) is printed at lightning checker's side, like 
```
"found PiTR log streaming task(s): [br_name],\n"+
		"found CDC capture(s): clusterID: <nil> captureID(s): [f14cb04d-5ba1-410e-a59b-ccd796920e9d],\n"+
		"local backend is not compatible with them. Please switch to tidb backend then try again."
```
no need to repeat again